renku/models/datasets.py

```diff
@@ -340,7 +340,6 @@ def _convert_keyword(keywords):
 @jsonld.s(
     type='schema:Dataset',
     context={
-        'added': 'schema:dateCreated',
```

Comment:
Why is this removed?

Reply:
because having two mappings to `schema:dateCreated` is not possible and I kept the one coming from `Dataset`. We should find another solution for this value in `DatasetFile` but I don't know at the moment what the correct term would be.

Reply:
Just keep in mind that if triples are not generated for `dateCreated` then it's probably the bug that I saw the other day about `isBasedOn`. You need to re-add this here in that case.

Reply:
ah no, they were being created, that was the problem - the ones from the `Dataset` itself were not, however, because in the `context` you can apparently only map a term once - in this case `dateCreated` was mapped to `created` and `added.

Reply:
but you're right - if at a later point we actually want to export the `added` information, it needs to be added above 👍